src/components/Toast/Toast.css

```diff
@@ -0,0 +1,59 @@
+:root{
+    --modal-bg:rgba(0,0,0,0.5);
+}
+
+/* Toast Css */
+
+.toast-container-top,
+.toast-container-bottom,
+.toast-dismiss{
+    padding:1rem 1rem;
+    color: var(--light-gray-2);
+    font-size: 0.9rem;
+    display: none;
+    z-index:1;
+    position: fixed;
+}
+.toast-dismiss{
+    top: 5rem;
+}
+#dismiss{
+    padding-left: 2rem;
+} 
+
+
+/* Modal Css */
+
+.modal {
+    position: fixed;
+    display: none;
+    width: 100%;
+    justify-content: center;
+    align-items: center;
+    height: 100%;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
```

Comment:
Suggestion: Keep similar style properties together for better comprehension and readability.
For instance, position: fixed along with the top, left, right and bottom properties together; width and height together and so on.